app/src/org/commcare/fragments/personalId/PersonalIdPhoneFragment.java

```diff
@@ -268,8 +267,9 @@ public void onTokenReceived(@NotNull String requestHash,
 
                     @Override
                     public void onTokenFailure(@NotNull Exception exception) {
-                        onConfigurationFailure(AnalyticsParamValue.START_CONFIGURATION_INTEGRITY_DEVICE_FAILURE,
-                                integrityTokenApiRequestHelper.getErrorForException(requireActivity(), exception));
+                        FirebaseAnalyticsUtil.reportPersonalIdConfigurationFailure(
+                                AnalyticsParamValue.START_CONFIGURATION_INTEGRITY_DEVICE_FAILURE);
```

Comment:
thoughts on removing this analytic now completely given we are not really failing the configuration flow anymore here and we are getting more wider reporting from the heartbeat anyway ? (Afraid that this will confuse anyone looking at those analytics)